ansible/roles/postgresql-data-update/tasks/main.yml

```diff
@@ -44,3 +46,21 @@
   become_user: "{{ enc_postgres_user }}"
   vars:
     ansible_ssh_pipelining: true
+
+- name: Create the schema for uci transformer-service
+  postgresql_db:
```

Comment:
@maheshkumargangula These run on VM and have slightly different synatx. Please refer to an exisitng task in this file. Example -`login_unix_socket`, `ansible_ssh_pipelining`
Please use the exact same synatx as one of the tasks in this file.